contribs/gmf/src/services/themesservice.js

```diff
@@ -54,13 +58,13 @@ gmf.Themes = function($http, treeUrl, isThemePrivateUrl) {
    * @type {string}
    * @private
    */
-  this.treeUrl_ = treeUrl;
+  this.treeUrl_ = '';
 
   /**
    * @type {string}
    * @private
    */
-  this.isThemePrivateUrl_ = isThemePrivateUrl;
+  this.isThemePrivateUrl_ = '';
```

Comment:
Luxembourg doesn't use this service yet.
If gmf2 doesn't support private theme url, then we need to remove it.
If a project like luxembourg want to use it, they need to add custom code in their own sources to handle it.